packages/block-library/src/gallery/block.json

```
@@ -43,6 +43,9 @@
 		},
 		"ids": {
 			"type": "array",
+  			"items": {
```

There's an extra tab here.
```suggestion
  		"items": {
```